code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -76,6 +76,16 @@
 	new /obj/item/weapon/reagent_containers/hypospray/medipen(src)
 	return
 
+/obj/item/weapon/storage/box/security
+
+/obj/item/weapon/storage/box/security/New()
+	..()
+	contents = list()

```

Comment:
I really only just used the same code used for the engineer internal box. If its that big a problem I'm sure someone else will figure out a alternative fix.